RapidReact/src/main/java/frc/robot/Robot.java

```diff
@@ -340,15 +330,24 @@ private void configureButtonBindings() {
       buttons.lowShoot.whenPressed(shooterSubsystem::shootLow);
       buttons.lowShoot.whenReleased(shooterSubsystem::stopShoot);
 
-      buttons.hubShoot.whenPressed(() -> {
-        shooterSubsystem.shootTop();
+      buttons.hubSpinUp.whenPressed(() -> {
+        shooterSubsystem.spinUpTop();
       });
-      buttons.hubShoot.whenReleased(() -> {
+      buttons.hubSpinUp.whenReleased(() -> {
```

Comment:
Do you want the operator to have to hold down a button continuously when the shooter is running, then press another to fire? Might be awkward...